jenkins/job-configs/kubernetes-jenkins/kubernetes-e2e-gce-gci.yaml

```diff
@@ -102,6 +102,40 @@
                 export GINKGO_TEST_ARGS="--ginkgo.focus=\[Serial\]|\[Disruptive\] \
                                          --ginkgo.skip=\[Flaky\]|\[Feature:.+\]"
                 export PROJECT="e2e-gce-gci-ci-serial"
+        - 'release-1.4':  # kubernetes-e2e-gce-gci-ci-release-1.4
+            description: 'Runs all non-slow, non-serial, non-flaky, tests on GCE with GCI images in parallel on the release-1.4 branch.'
+            timeout: 50
+            trigger-job: 'kubernetes-build-1.4'
+            job-env: |
+                export JENKINS_PUBLISHED_VERSION="ci/latest-1.4"
+                # To qualify GCI milestone 55 for k8s release-1.4.
+                export JENKINS_GCI_HEAD_IMAGE_FAMILY="gci-canary"
```

Comment:
This case is special though (1.4 is already cut and we are switching GCI milestones). Normally we won't setup a CI (tip for both k8s and GCI) for a k8s release branch - the latest GCI milestone should better be tested in k8s master, i.e., before a release branch is cut.